Frontend/ui-driver/src/Screens/SubscriptionScreen/View.purs

```diff
@@ -1595,9 +1612,9 @@ dueOverViewCard push state isManual =
         , onClick (\action -> do
             _<- push action
             _ <- JB.openUrlInApp $ case getValueToLocalNativeStore LANGUAGE_KEY of
-                              "EN_US" -> "https://www.youtube.com/playlist?list=PL4AEiRR3V7kHcg2-fgzvDXDqWihZD9mTK"
-                              "KN_IN" -> "https://www.youtube.com/playlist?list=PL4AEiRR3V7kHcg2-fgzvDXDqWihZD9mTK"
-                              _ -> "https://www.youtube.com/playlist?list=PL4AEiRR3V7kHcg2-fgzvDXDqWihZD9mTK"
+                              "EN_US" -> state.data.config.myPlanYoutubeLink
```

Comment:
directly pass `state.data.config.myPlanYoutubeLink`

No need for case match now